packages/jobs/package.json

```diff
@@ -29,7 +30,7 @@
         "dotenv": "^16.0.3",
         "fs-extra": "^11.1.1",
         "js-yaml": "^4.1.0",
-        "knex": "^2.4.2",
+        "knex": "3.1.0",
```

Comment:
it is a major upgrade. Are we fully confident upgrading doesn't break anythin?

Reply:
the main breaking change is the drop of Node16 and some small types fixes 
https://github.com/knex/knex/releases/tag/3.0.0

But I would love to test on staging before merging